test/sequential/test-inspector-contexts.js

```diff
@@ -22,7 +22,13 @@ async function testContextCreatedAndDestroyed() {
     const mainContextPromise =
         notificationPromise('Runtime.executionContextCreated');
 
-    session.post('Runtime.enable');
+<<<<<<< HEAD
```

Comment:
@Trott looks like you've accidentally committed merge conflicts.

Reply:
/facepalm....